ivy_tests/conftest.py

```diff
@@ -55,7 +55,7 @@ def pytest_terminal_summary(terminalreporter):
         return
 
     passed_ratio = 1 - (session.testsfailed / session.testscollected)
-    text = " {:.1%} of {} passed ".format(passed_ratio, session.testscollected)
+    text = f" {passed_ratio:.1%} of {session.testscollected} passed "
```

Comment:
I think the `%` sign should be removed from here

Reply:
@vedpatwardhan sir, I think it should be present. Please correct me if I am wrong.

Reply:
sorry my bad, yeah we should retain the `%` 👍